src/components/header/header.tsx

```diff
@@ -18,55 +18,37 @@ const getTitle = () => {
   return title;
 };
 
-class Header extends React.Component<object, { title?: string }> {
-  constructor(public props: object) {
-    super(props);
-    this.state = {
-      title: getTitle(),
-    };
-  }
+const Header = () => {
+  const [titleValue, setTitle] = useState(getTitle());
 
-  private changeTitle(title: string): void {
-    this.setState({
-      title: title,
-    });
-  }
+  const changeTitle = (title: string): void => {
+    setTitle(title);
+  };
 
-  render() {
-    return (
-      <header className={classes.header}>
-        <nav>
-          <ul>
-            <li>
-              <NavLink
-                to={pageData.pagePath[0]}
-                onClick={() => this.changeTitle(pageData.pageTitle[0])}
-              >
-                Main
-              </NavLink>
-            </li>
-            <li>
-              <NavLink
-                to={pageData.pagePath[1]}
-                onClick={() => this.changeTitle(pageData.pageTitle[1])}
-              >
-                About Us
-              </NavLink>
-            </li>
-            <li>
-              <NavLink
-                to={pageData.pagePath[2]}
-                onClick={() => this.changeTitle(pageData.pageTitle[3])}
-              >
-                Forms
-              </NavLink>
-            </li>
-          </ul>
-        </nav>
-        <h2>{this.state.title}</h2>
-      </header>
-    );
-  }
-}
+  return (
+    <header className={classes.header}>
+      <nav>
+        <ul>
+          <li>
+            <NavLink to={pageData.pagePath[0]} onClick={() => changeTitle(pageData.pageTitle[0])}>
```

Comment:
better for each page and title create a separate enum or object with values:
```
const Pages = {
  main: '/',
  aboutUs: '/aboutus',
  forms: '/forms'
}
```
the same for titles and form values, it will be much more readable